app/src/main/res/layout/popup.xml

```diff
@@ -6,16 +6,26 @@
     android:layout_height="match_parent"
     android:gravity="bottom">
 
+    <TextView
+        android:id="@+id/textView4"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
+        android:text="@string/journey_details"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
```

Comment:
Could possibly be a little bigger and stand out more if this is intended to be a "title"

Reply:
Ok